README.md

```diff
@@ -1,147 +1,35 @@
-![logo](logo.png)
---
+# [Jekyll Dash](https://github.com/bitbrain/jekyll-dash) without plugins  
 
-A dark and light theme for Jekyll, inspired by Dash UI for Atom. 🌒☀
+Jekyll Dash is a Jekyll theme made by bitbrain.
 
-[![Build Status](https://img.shields.io/travis/bitbrain/braingdx/master.svg?logo=travis&style=flat-square)](https://travis-ci.org/bitbrain/jekyll-dash)
```

Comment:
why did you remove this? :(

Reply:
Sorry, I upload the wrong one. I created three or four README and I didn't check it before and after I uploaded. I will upload the new one, which I just add something at the bottom of your original README.

Sorry for this mistake.